CloudinaryDotNet/ApiShared.Internal.cs

```diff
@@ -478,6 +499,36 @@ private void SetChunkedEncoding(HttpRequestMessage request)
             }
         }
 
+        private void SetUserPlatform(HttpHeaderValueCollection<ProductInfoHeaderValue> userAgentHeader)
+        {
+            Console.WriteLine($"UserPlatform: [{UserPlatform}] ======");
+            var up = UserPlatform?.Trim();
+            if (string.IsNullOrEmpty(up))
+            {
+                return;
+            }
+
+            var upp = up.Split('/');
+            var productName = GetElement(0);
+            if (string.IsNullOrEmpty(productName))
+            {
+                return;
+            }
+
+            var productVersion = GetElement(1);
+            if (string.IsNullOrEmpty(productVersion))
+            {
+                productVersion = "0.1";
```

Comment:
"0.1" ? is there any special meaning?

Reply:
@const-cloudinary No special meaning here, just a default value.